depthai_helpers/arg_manager.py

```diff
@@ -84,4 +84,5 @@ def parse_args():
                         help="Mono cam fps: max 60.0 for H:720 or H:800, max 120.0 for H:400. Default: %(default)s")
     parser.add_argument('-cb', '--callback', type=Path, default=project_root / 'callbacks.py', help="Path to callbacks file to be used. Default: %(default)s")
     parser.add_argument("--openvino_version", type=str, choices=openvino_versions, help="Specify which OpenVINO version to use in the pipeline")
+    parser.add_argument("--count", type=str, help="Count and display the number of specified objects on the frame. You have to specify the object (label) you want to count.")
```

Comment:
I would add that the label can be either a label id or label name (if defined labels in nn config file)